content/posts/visual-studio-code-setup.md

```diff
@@ -0,0 +1,142 @@
+
+### Code spell checker
+
+![Code spell checker in the visual studio code store](./images/code-spell-checker.png)
+
+I’m don’t spell well most of the time. This [extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) is a life saver. Plus it helps find little bugs due to typos.
+
+### ESLint
+
+![ESLint extension in the visual studio code store](./images/eslint.png)
+
+[ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) is an essential extension for any JavaScript based project. Getting all the settings to work are a pain in the ass sometimes, but I couldn't live without it.
+
+### Headwind
+
+![Headwind in the visual studio code store](./images/headwind.png)
+
+This [extension](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) helps me align my tailwind classes in the same order without having to thing about it. It doesn’t seem like much, but the consistency of the order without having to think about is awesome.
+
+### Tailwind CSS intelliSence
+
+![Tailwind CSS intelliSence in the visual studio code store](./images/tailwind-css-intellisense.png)
+
+[IntelliSence](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) is a great feature of VSCode. Why not have it know about my Tailwind classes? Very useful for remembering the number differences for spacing and colors.
+
+### Toggle quotes
+
+![Toggle quotes in the visual studio code store](./images/toggle-quotes.png)
+
+This [extension](https://marketplace.visualstudio.com/items?itemName=BriteSnow.vscode-toggle-quotes) seems like it would be native to VSCode along with all the other text transformation tools. I can easily switch between single quotes, double quotes, and back ticks. I even like the default keyboard shortcut `control + ‘`.
+
+### Vetur
+
+![Vetur is the perfect Vue integration extension](./images/vetur.png)
+
+[Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur) gives you easy integration with all things Vue in VSCode. It's an essential extension for working with Vue projects. What are you waiting for, download this package right now!
+
+### Vscode-icons
+
+![Example of vscode-icons for vue, javascript, etc](./images/vscode-icons.png)
+
+I've used the [vscode-icon](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons) set since I started using Visual Studio Code. I'm so use to them now, I don't want to switch.
+
+### Vue vscode snippets
+
+![Vue vscode snippets is the best collection of snippets for the Vue.js language](./images/vue-vscode-snippets.png)
+
+The [best snippet](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-snippets) package for Vue.js. I sometimes forget the format of certain features because this package has so many good snippets that I've forgot the right syntax from memory 😅
```

Comment:
the "I sometimes forget..." sentence runs a bit. maybe just "I sometimes forget the format of certain features because this package has so many good snippets."?

also missing a period.